enterprise/server/raft/testutil/testutil.go

```diff
@@ -139,7 +139,10 @@ func (sf *StoreFactory) NewStore(t *testing.T) *TestingStore {
 	ts.Store = store
 
 	t.Cleanup(func() {
-		store.Stop(context.TODO())
+		ctx := context.Background()
```

Comment:
Can we use env.GetServerContext here instead?

For local testing, the shutdown timeout is like 3 seconds

Reply:
we don't have a env here. changing the timeout from 10s to 3 seconds